questionnaire/utils/getOwnerOrigin/index.js

```diff
@@ -0,0 +1,14 @@
+'use strict';
+
+function getOwnerOrigin(request, interalIpArray) {
```

Comment:
I'm thinking that you shouldn't pass in the IPs and just get them from the env vas at this point. Saves having to pass them in, plus it's only tis module that is concerned with them anyway.

At the very least, specifying a default in the function parameters is the way to go here. not needing to be explicitly passed in, yet modifiable. However, this function is concerned with checking against a specific set of IPs, but the implementation allows for checking against "whatever is passed in".